projects/envoy/build.sh

```diff
@@ -41,7 +41,7 @@ if [ "$SANITIZER" = "undefined" ]
 then
   # Bazel uses clang to link binary, which does not link clang_rt ubsan library for C++ automatically.
   # See issue: https://github.com/bazelbuild/bazel/issues/8777
-  echo "--linkopt=$(find $(llvm-config --libdir) -name libclang_rt.ubsan_standalone_cxx-x86_64.a | head -1)"
+  echo "--linkopt=$(find $(llvm-config --libdir) -name libclang_rt.ubsan_standalone_cxx.a | grep x86_64 | head -1)"
```

Comment:
FWIW, I think this could be simplified to:
```suggestion
  echo "--linkopt=$(clang -print-file-name=libclang_rt.ubsan_standalone_cxx.a)"
```

See for example:

```console
$ docker run -it gcr.io/oss-fuzz-base/base-builder sh -c "clang -print-file-name=libclang_rt.ubsan_standalone_cxx.a"
/usr/local/lib/clang/18/lib/x86_64-unknown-linux-gnu/libclang_rt.ubsan_standalone_cxx.a
```

Reply:
Thanks!
I'll test it out (it will take ~1 hour to build the fuzzers), and if it works I'll update the PR.

Reply:
Thanks for the recommendation, it seems to be working!
I've updated the script accordingly.